### PR TITLE
Change from Editor to Creator for Save / Apply on new events.

### PR DESCRIPTION
**Pull request by tonypartridge**

### CODE REVIEW

**Comment by GeraintEdwards** [comment]
Does apply work with this change?  Since if you can't edit your own events then it should fail when it tries to reopen the edit page.

**Comment by tonypartridge** [comment]
Thats a good point! let me re-adjust this to hide apply.

**Comment by tonypartridge** [comment]
Updated to put apply into only shown on editown is allowed.

File: component/site/views/icalevent/view.html.php [hunk 1]
```diff
@@ -97,11 +97,13 @@ function edit($tpl = null)
 				$juser = JFactory::getUser();
 				$canEditOwn = $juser->authorise('core.edit.own', 'com_jevents');
 			}
-			if (JEVHelper::isEventEditor() || $canEditOwn)
+			if (JEVHelper::canCreateEvent() || $canEditOwn)
 			{
-                $this->toolbarButton("icalevent.apply", 'apply', 'apply', 'JEV_SAVE', false);
 				$this->toolbarButton("icalevent.save", 'save', 'save', 'JEV_SAVE_CLOSE', false);
 			}
+			if ($canEditOwn) {
```
Comment (GeraintEdwards) [inline component/site/views/icalevent/view.html.php]: should really be 

if (JEVHelper::isEventEditor() || $canEditOwn)

Comment (tonypartridge) [inline component/site/views/icalevent/view.html.php]: Agreeded, updated.

**Comment by tonypartridge** [comment]
I thought this had been merged Geraint :-(.

**Pull request closed without merging**